src/Clock/GenericClock.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Clock;
+
+use DateTimeImmutable;
+use DateTimeInterface;
+
+final class GenericClock implements Clock
+{
+    public function now(): DateTimeInterface
+    {
+        return new DateTimeImmutable('now');
+    }
+
+    public function time(): int
+    {
+        return time();
+    }
+
+    public function sleep(int $seconds): void
+    {
+        sleep($seconds);
```

Comment:
Isn't sleep only useful for testing? That's an indication it shouldn't be on the interface but rather on `MockClock`

Reply:
I've definitely used `sleep` for non-testing things before in CLI. Most clock implementations I've seen tend to include it (e.g., [Symfony](https://github.com/symfony/symfony/blob/7.0/src/Symfony/Component/Clock/Clock.php)).

Reply:
Oh, TIL. Then let's keep it 👍